app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    line = command.split()
```

Comment:
Use unpacking for this case, but before chach that `len(command) == 3`

Reply:
Strange checks:
If len(line) != 3 then go inside - wrong!
if line[0] != "cp" then do copy - wrong!
if line[1] == line[2] then  do copy - wrong!


All checks are wrong! Must be all 3 true len == 3, the command is cp, files names are different